.jenkins_quicktest

```diff
@@ -1,34 +1,38 @@
 #!/bin/bash
 set -e
 set -o pipefail
+export IBAMR_DIR=$PWD
+export IBAMR_ARCH=build
+export PETSC_DIR=$IBAMR_DIR/$IBAMR_ARCH/PETSC
+export PETSC_ARCH=linux-debug
 
-# export the following environment variables
-# before executing this script in the jenkins
-# execute shell
+build_petsc() {
+    mkdir -p $PETSC_DIR && git clone -b knepley/ibamr https://bitbucket.org/deleeke/petsc $PETSC_DIR
+    cd $PETSC_DIR
```

Comment:
What petsc should this be cloning? I don't think mine is best.

Reply:
This branch would make things more stable since its completely under your control.

Should I update it to the latest master?